httpbp/middlewares.go

```diff
@@ -422,6 +421,8 @@ func recordStatusCode(counters counterGenerator) Middleware {
 // RecordStatusCode should generally not be used directly, instead use the
 // NewBaseplateServer function which will automatically include RecordStatusCode
 // as one of the Middlewares to wrap your handlers in.
+//
+// Deprecated: This is deprecated with statsd metrics.
```

Comment:
We should have waited for this cleanup before doing https://github.com/reddit/baseplate.go/pull/573 😅